nix/cardano-services/deployments/chain-history.nix

```diff
@@ -0,0 +1,58 @@
+{
+  config,
+  values,
+  lib,
+  utils,
+  chart,
+  ...
+}: {
+  providers.chain-history-provider = {
+    inherit (values.cardano-services) image;
+    args = ["start-provider-server"];
+    port = 3000;
+    metricsPath = "${values.cardano-services.httpPrefix}/metrics";
+
+    livenessProbe = {
+      timeoutSeconds = 5;
+      httpGet = {
+        path = "${values.cardano-services.httpPrefix}/health";
+        port = 3000;
+      };
+    };
+
+    env = {
+      NETWORK = values.network;
+      OGMIOS_SRV_SERVICE_NAME = values.backend.ogmiosSrvServiceName;
+      LOGGER_MIN_SEVERITY = values.cardano-services.loggingLevel;
+      ENABLE_METRICS = "true";
+      DISABLE_STAKE_POOL_METRIC_APY = "true";
+      PAGINATION_PAGE_SIZE_LIMIT = "5500";
+      SERVICE_NAMES = "chain-history";
```

Comment:
i assume this line is the only change compared to the rest of env variables from monolithic backend?

Reply:
I just updated the env to match backend yes